spec.html

```diff
@@ -28857,10 +28857,18 @@ <h1>
         </dl>
         <emu-alg>
           1. Assert: If _direct_ is *false*, then _strictCaller_ is also *false*.
-          1. If _x_ is not a String, return _x_.
+          1. If _x_ is an Object, then
+            1. Let _code_ be HostGetCodeForEval(_x_).
+            1. If _code_ is ~unknown~ return _x_.
```

Comment:
Maybe if the "code-like" terminology is going to persist, do

```suggestion
            1. If _code_ is ~not-code-like~ return _x_.
```

Reply:
The name "code-like" is currently not referenced anywhere (it was previously the name of the slot), not even in the TT spec. Maybe just `~no-code~`?

Reply:
Yeah I'm happy with that. If there's terminology used in the trusted types spec that you'd like to appropriate, that would also work for me.